tests/framework/installer/ceph_manifests.go

```diff
@@ -584,7 +584,7 @@ spec:
   - replicated:
       size: 1
   metadataServer:
-    activeCount: 1
+    activeCount: 2
```

Comment:
I changed the activeCount here to `2` as you suggested, and I would expect that the FS should now have 4 total mdses: 2 active and 2 standby.

```
2018-10-04 19:31:10.875621 I | testutil: waiting for pod(s) with label rook_file_system=upgrade-test-fs in namespace upgrade-ns to be running. status=Pending, running=0/2, err=<nil>
...
2018-10-04 19:33:45.985035 E | testutil: Expecting 4 number of pods for rook-ceph-mds app, found 2 
```